### ISSUE TITLE
Change text area

### ISSUE THREAD

**Issue by aidenmitchell** [OrangeJedi/Aerial#24]
**Is your feature request related to a problem? Please describe.**
The video description text is great, but it doesn't span the full width of the screen, it wraps about a  1/3 of the way through.

**Describe the solution you'd like**
Text uses full width of display when necessary.

**Describe alternatives you've considered**
Changing text size.

**Additional context**
![image](https://user-images.githubusercontent.com/30846409/79711757-9cb74200-827d-11ea-8dd5-8f6e94e1718c.png)

**Comment by OrangeJedi** [comment]
I believe this is related to having the 9 text areas; Aerial is cutting off the text (unnecessarily) to avoid any possible collisions. It should be any easy fix, it does look better all in one line. I might spend some more time on it though, trying to make it more customizable.

In the meantime, you can manually fix it on your version by opening the folder where Aerial is installed and opeing `\resources\app\web\screensaver.css` and replacing 
```
.textDisplayArea {
    padding: 2%;
    width: 50%;
}
```
```
.textDisplayArea {
    padding: 2%;
}
```
at the end of the file.

I'll update this issue with any other progress.

**Comment by aidenmitchell** [comment]
Oh cool, it's CSS! Thanks.

**Comment by aidenmitchell** [comment]
Not sure where I installed Aerial though, when I installed it I just right-clicked the .scr and clicked Install. How could I find the `\resources` folder?

**Comment by OrangeJedi** [comment]
If you downloaded the folder it should be in the directory.  If you just downloaded the .scr file then the `\resources` folder is complied into .scr and you can't assess it.

**Comment by OrangeJedi** [comment]
Fixed in [v0.5.3](https://github.com/OrangeJedi/Aerial/releases/tag/v0.5.3-beta). Let me know if you have any trouble. You will need to change a 'max width' selector in the text settings to 100% for the text to span the screen.
